packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -93,58 +96,51 @@ export class Relayer {
           continue;
         }
         try {
-          await this._relayPendingDeposit(l1Token, relayableDeposit);
+          relayTransactions.push(await this._generateRelayTransactionForPendingDeposit(l1Token, relayableDeposit));
         } catch (error) {
-          this.logger.error({
-            at: "InsuredBridgeRelayer#Relayer",
-            message: "Unexpected error processing deposit",
-            error,
-          });
+          this.logger.error({ at: "AcrossRelayer#Relayer", message: "Unexpected error processing deposit", error });
         }
       }
+      try {
```

Comment:
@nicholaspai wrapping the `_processTransactionBatch` that could error. With this the bot should never halt to block another action.